src/schema/vacationRequest.ts

```diff
@@ -1,15 +1,26 @@
 import { Type } from "@sinclair/typebox";
+import {VacationRequestStatuses} from "@generated/client/model/vacationRequestStatuses";
 
 /**
- * Schema for the Questionnaire table
+ * Schema for each individual status entry in a vacation request
  */
+const vacationRequestStatusSchema = Type.Object({
```

Comment:
So the spec will need updating again, we need to remove the vacationRequestStatuses endpoint and update the schema.
Also remember if any of these attributes are optional it should be described as such here.

And related to this, it looks like at least some VacationRequest attributes are described as optional in the spec and I don think some of them should be optional e.g. startDate, createdBy. I should have spotted it previously but might as well get it updated now.